<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>625</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(223, 223, 223)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="MenuLabel">
    <property name="geometry">
     <rect>
      <x>1090</x>
      <y>30</y>
      <width>64</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>MENU</string>
    </property>
   </widget>
   <widget class="QLabel" name="ChattingMenuLabel">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>381</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>&quot;Chatting Menu&quot;</string>
    </property>
   </widget>
   <widget class="QLabel" name="IDLabel">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>20</y>
      <width>70</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>&quot;ID BOX&quot;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ChattingLineEdit">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>530</y>
      <width>450</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget_Chat">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>70</y>
      <width>630</width>
      <height>430</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget_Users">
    <property name="geometry">
     <rect>
      <x>755</x>
      <y>70</y>
      <width>160</width>
      <height>430</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
